ceos Startup config test fails

Running and startup config is identical the only difference is the command info output

``` python
E       AssertionError: assert False == True
E        +  where False = bool('! Command: s...000\n!\nend\n' == '! Command: s...000\n!\nend\n'
E           - ! Command: show running-config
E           ?                   ^^^^^
E           + ! Command: show startup-config
E           ?                 +++ + ^
E           + ! Startup-config last modified at Fri Jul 12 15:59:25 2024 by root
E             ! device: fra05-pod1-spine2 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))
E             !...
E           
E           ...Full output truncated (210 lines hidden), use '-vv' to show)
```

https://nuts.readthedocs.io/en/latest/testbundles/alltestbundles.html#configuration-startup-vs-running

--- a/nuts/base_tests/napalm_get_config.py
+++ b/nuts/base_tests/napalm_get_config.py
@@ -1,5 +1,6 @@
 """Query config of a device."""
 
+import re
 from typing import Dict, Callable, List, Any
 
 import pytest
@@ -34,10 +35,7 @@ class TestNapalmConfig:
     def test_startup_equals_running_config(
         self, single_result, startup_equals_running_config
     ):
-        assert (
-            bool(
-                single_result.result["config"]["startup"]
-                == single_result.result["config"]["running"]
-            )
-            == startup_equals_running_config
-        )
+        pattern = r"! Command: show (startup|running)-config\n(! Startup-config last modified at .* by .*\n)?"
+        startup = re.sub(pattern, "", single_result.result["config"]["startup"])
+        running = re.sub(pattern, "", single_result.result["config"]["running"])
+        assert bool(startup == running) == startup_equals_running_config

--- a/nuts/base_tests/napalm_get_config.py
+++ b/nuts/base_tests/napalm_get_config.py
@@ -35,7 +35,10 @@ class TestNapalmConfig:
     def test_startup_equals_running_config(
         self, single_result, startup_equals_running_config
     ):
-        pattern = r"! Command: show (startup|running)-config\n(! Startup-config last modified at .* by .*\n)?"
+        pattern = (
+            r"! Command: show (startup|running)-config\n"
+            r"(! Startup-config last modified at .* by .*\n)?"
+        )
         startup = re.sub(pattern, "", single_result.result["config"]["startup"])
         running = re.sub(pattern, "", single_result.result["config"]["running"])
         assert bool(startup == running) == startup_equals_running_config

--- a/tests/base_tests/test_napalm_get_config.py
+++ b/tests/base_tests/test_napalm_get_config.py
@@ -305,8 +305,17 @@ nornir_raw_result_s1 = {
 
 nornir_raw_result_s2 = {
     "config": {
-        "startup": "! Command: show startup-config\n! Startup-config last modified at Fri Jul 12 15:59:25 2024 by root\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
-        "running": "! Command: show running-config\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
+        "startup": "! Command: show startup-config\n"
+        "! Startup-config last modified at Fri Jul 12 15:59:25 2024 by root\n"
+        "! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M"
+        "(engineering build))\n"
+        "!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n"
+        "   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
+        "running": "! Command: show running-config\n"
+        "! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M"
+        "(engineering build))\n"
+        "!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n"
+        "   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
     }
 }
 

--- a/tests/base_tests/test_napalm_get_config.py
+++ b/tests/base_tests/test_napalm_get_config.py
@@ -303,11 +303,24 @@ nornir_raw_result_s1 = {
     }
 }
 
+nornir_raw_result_s2 = {
+    "config": {
+        "startup": "! Command: show startup-config\n! Startup-config last modified at Fri Jul 12 15:59:25 2024 by root\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
+        "running": "! Command: show startup-config\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
+    }
+}
+
 
 config_s1 = SelfTestData(
     name="s1",
     nornir_raw_result=nornir_raw_result_s1,
     test_data={"host": "S1", "startup_equals_running_config": True},
+)
+
+config_s2 = SelfTestData(
+    name="s2",
+    nornir_raw_result=nornir_raw_result_s2,
+    test_data={"host": "S2", "startup_equals_running_config": True},
 )
 
 
@@ -324,7 +337,7 @@ def general_result(timeouted_multiresult):
 
 
 @pytest.fixture(
-    params=[config_s1],
+    params=[config_s1, config_s2],
     ids=lambda data: data.name,
 )
 def selftestdata(request):

--- a/tests/base_tests/test_napalm_get_config.py
+++ b/tests/base_tests/test_napalm_get_config.py
@@ -306,7 +306,7 @@ nornir_raw_result_s1 = {
 nornir_raw_result_s2 = {
     "config": {
         "startup": "! Command: show startup-config\n! Startup-config last modified at Fri Jul 12 15:59:25 2024 by root\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
-        "running": "! Command: show startup-config\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
+        "running": "! Command: show running-config\n! device: fra05-pod1-leaf1 (cEOSLab, EOS-4.28.9M-33818481.4289M (engineering build))\n!\nno aaa root\n!\nusername admin privilege 15 role redistribute connected\n   network 0.0.0.0/0 area 0.0.0.0\n   max-lsa 12000\n!\nend\n'",
     }
 }
 

